changes/1474.bugfix.md

```diff
@@ -0,0 +1,2 @@
+- Implement `GuildThreadChannel.edit`
+- Update `RESTClient.edit_channnel` to support applied tags on forum threads
```

Comment:
This one would not fall as a bugfix, but as a feature rather

Reply:
👍